content/en/resources/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Resources
+menu: {main: {weight: 50}}
+description: Learn more from our resources
+
```

Comment:
Avoid blank lines in the the page front matter, in this file and all other files:

```suggestion
```